modules/test/playwright/tests/configuration-admin-web/instanceSettings.spec.ts

```diff
@@ -6,26 +6,36 @@
 import {expect, mergeTests} from '@playwright/test';
 import {readFile} from 'fs/promises';
 
+import {accessibilityMenuPagesTest} from '../../fixtures/accessibilityMenuPagesTest';
+import {instanceSettingsPagesTest} from '../../fixtures/instanceSettingsPagesTest';
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
 import {loginTest} from '../../fixtures/loginTest';
-import {InstanceSettingsPage} from '../../pages/configuration-admin-web/InstanceSettingsPage';
+import {siteSettingsPagesTest} from '../../fixtures/siteSettingsPagesTest';
 import {UsersAndOrganizationsPage} from '../../pages/users-admin-web/UsersAndOrganizationsPage';
 import getRandomString from '../../utils/getRandomString';
 
-export const test = mergeTests(loginTest());
-
-test('Asserts that a user can export a configuration', async ({page}) => {
+export const test = mergeTests(
+	accessibilityMenuPagesTest,
+	instanceSettingsPagesTest,
+	isolatedSiteTest,
+	loginTest(),
+	siteSettingsPagesTest
+);
+
+test('Asserts that a user can export a configuration', async ({
+	instanceSettingsPage,
+	page,
+}) => {
 	const emailDomainValidationSwitcher = page.getByRole('switch', {
 		name: 'Enable Email Domain Validation',
 	});
 
-	const instanceSettingsPage = new InstanceSettingsPage(page);
-
 	await instanceSettingsPage.goToInstanceSetting('Accounts', 'Email Domains');
 
 	try {
 		await emailDomainValidationSwitcher.check();
 
-		await instanceSettingsPage.saveButton.click();
+		await instanceSettingsPage.saveAndWaitForAlert({});
```

Comment:
Can we add defaults to `saveAndWaitForAlert` so that we don't have to pass an empty object as the param?